infrastructure/router/router.go

```diff
@@ -0,0 +1,32 @@
+package router
+
+import (
+	"net/http"
+	"strconv"
+	"taskmanager/interface/controllers"
+
+	"github.com/gorilla/mux"
+)
+
+// NewRouter defines the endpoints of the Application
+func NewRouter(muxRouter *mux.Router, c controllers.AppController) *mux.Router {
+	muxRouter.HandleFunc("/tasks", func(rw http.ResponseWriter, r *http.Request) {
+		rw.Header().Add("content-type", "application/json")
+		c.GetTasks(rw)
```

Comment:
I guess you need to validate the error that this function returns. What do you think?

Reply:
Yes, I missed the case when something failed, I am adding an internal error response for these cases